configure.ac

```diff
@@ -1592,6 +1592,69 @@ AC_SUBST(OPENTHREAD_ENABLE_ECDSA)
 AM_CONDITIONAL([OPENTHREAD_ENABLE_ECDSA], [test "${enable_ecdsa}" = "yes"])
 AC_DEFINE_UNQUOTED([OPENTHREAD_ENABLE_ECDSA], [${OPENTHREAD_ENABLE_ECDSA}], [Define to 1 if you want to enable ECDSA support])
 
+#
+# Support for OQPSK modulation in 915MHz band
+#
+
+AC_MSG_CHECKING([whether to support OQPSK modulation in 915MHz band])
+AC_ARG_ENABLE(radio_scheme_915mhz_oqpsk,
```

Comment:
Because these build options may cause incompatibility with existing Thread 1.1.1, I would prefer to move these to [src/core/openthread-core-default-config.h](https://github.com/openthread/openthread/blob/master/src/core/openthread-core-default-config.h).